xml_converter/doc/position/position.md

```diff
@@ -11,16 +11,20 @@ components:
   type: Float32
   xml_fields: [XPos, PositionX]
   protobuf_field: "x"
+  examples: ["166.735", "1110.4", "-433.615", "-196.412"]
 
 - name: Y Position
   type: Float32
   xml_fields: [YPos, PositionY]
   protobuf_field: "y"
+  examples: ["81.727", "24.5189", "9.12564", "3.91473"]
```

Comment:
One of these should be negative since they are valid values.

Reply:
Done